service-onboarding-build-pack/Jenkinsfile

```diff
@@ -31,6 +31,10 @@ def Event_Status = [
 	'FAILED':'FAILED'
 ]
 @Field def region = ''
+@Field def g_svc_admin_cred_ID = 'SVC_ADMIN'
+@Field def auth_token = ''
+@Field def g_base_url = ''
+@Field def jq = '/usr/local/bin/jq'
```

Comment:
Is this required?

Reply:
Surya had requested to install jq in the jenkins server in one of the previous sprints.